enhancements/console/dynamic-plugins.md

```diff
@@ -493,6 +504,22 @@ None
 
 None
 
+### API Extensions
```

Comment:
Let's override the job instead of adding these sections per the guidance here:

https://github.com/openshift/enhancements/#template-updates